server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/settings/SettingsPage.java

```diff
@@ -17,68 +17,64 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarqube.pageobjects.settings;
+package org.sonarqube.qa.util.pageobjects.settings;
 
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Selenide;
 import com.codeborne.selenide.SelenideElement;
 import org.openqa.selenium.By;
 
-import static com.codeborne.selenide.Condition.cssClass;
-import static com.codeborne.selenide.Condition.exactValue;
-import static com.codeborne.selenide.Condition.exist;
-import static com.codeborne.selenide.Condition.visible;
-import static com.codeborne.selenide.Selenide.$;
-
 public class SettingsPage {
 
   public SettingsPage() {
-    $("#settings-page").shouldBe(visible);
+    Selenide.$("#settings-page").shouldBe(Condition.visible);
   }
 
   public SettingsPage assertMenuContains(String categoryName) {
-    $(".side-tabs-menu").$(By.linkText(categoryName)).shouldBe(visible);
+    Selenide.$(".side-tabs-menu").$(By.linkText(categoryName)).shouldBe(Condition.visible);
     return this;
   }
 
   public SettingsPage assertSettingDisplayed(String settingKey) {
-    $(".settings-definition[data-key='" + settingKey + "']").shouldBe(visible);
+    Selenide.$(".settings-definition[data-key='" + settingKey + "']").shouldBe(Condition.visible);
     return this;
   }
 
   public SettingsPage assertSettingNotDisplayed(String settingKey) {
-    $(".settings-definition[data-key='" + settingKey + "']").shouldNotBe(visible);
+    Selenide.$(".settings-definition[data-key='" + settingKey + "']").shouldNotBe(Condition.visible);
     return this;
   }
 
   public SettingsPage openCategory(String categoryName) {
-    $(".side-tabs-menu").$(By.linkText(categoryName)).click();
+    Selenide.$(".side-tabs-menu").$(By.linkText(categoryName)).click();
     return this;
   }
 
   public SettingsPage assertStringSettingValue(String settingKey, String value) {
-    $("input[name=\"settings[" + settingKey + "]\"]").shouldHave(exactValue(value));
+    Selenide.$("input[name=\"settings[" + settingKey + "]\"]").shouldHave(Condition.exactValue(value));
     return this;
   }
 
   public SettingsPage assertBooleanSettingValue(String settingKey, boolean value) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "value" to determine which action to take. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2301)